spring-content-s3/src/test/java/internal/org/springframework/content/s3/store/DefaultS3StoreImplTest.java

```diff
@@ -373,6 +374,148 @@ public S3ObjectId convert(CustomContentId entity) {
                         });
                     });
 				});
+				Context("#getResource(entity, propertyPath)", () -> {
+					JustBeforeEach(() -> {
+						try {
+							r = s3StoreImpl.getResource(entity, PropertyPath.from("content"));
+						}
+						catch (Exception e) {
+							this.e = e;
+						}
+					});
+
+					// the following context is exactly the same as for "#getResource(entity)" above
```

Comment:
I have a big chunk of test code duplication here. Is there a way to organize it better or it is not an issue?